gui/src/installer/view.rs

```diff
@@ -884,6 +886,236 @@ pub fn define_bitcoin<'a>(
+            .size(20)
+            .padding(10),
+        )
+        .spacing(10);
+
+    let col_prune = Column::new()
+        // TODO: Add warnings about space required and time for IBD.
+        .push(text("Prune amount (MB):").bold())
+        .push(
+            form::Form::new(PRUNE_DEFAULT.to_string().as_str(), prune, |msg| {
+                Message::SaveInternalBitcoindConfig(message::InternalBitcoindConfigEdited::Prune(
+                    msg,
+                ))
+            })
+            .warning("Please enter prune amount")
+            .size(20)
+            .padding(10),
+        )
+        .spacing(10);
+
+    let next_button = button::primary(None, "Next").width(Length::Fixed(200.0));
+    layout(
+        progress,
+        "Configure your Bitcoin node",
+        Column::new()
+            .push(col_rpc_port)
+            .push(col_p2p_port)
+            .push(col_prune)
+            .push(
+                Container::new(
+                    Column::new()
+                        .spacing(20)
+                        .align_items(Alignment::Center)
+                        .push(if rpc_port.valid && p2p_port.valid && prune.valid {
+                            next_button.on_press(Message::Next)
+                        } else {
+                            next_button
+                        })
+                        .align_items(Alignment::Center),
+                )
+                .padding(20),
+            )
+            .push_maybe(if file_saved {
+                Some(Container::new(
+                    Row::new()
+                        .spacing(10)
+                        .align_items(Alignment::Center)
+                        .push(icon::circle_check_icon().style(color::GREEN))
+                        .push(
+                            text(format!("Values saved to \"{}\"", conf_path)).style(color::GREEN),
+                        ),
+                ))
+            } else {
+                Some(Container::new(
+                    Row::new()
+                        .spacing(10)
+                        .align_items(Alignment::Center)
+                        .push(text(format!("Values will be saved to \"{}\"", conf_path))),
+                ))
+            }),
+        true,
+    )
+}
+
+pub fn start_internal_bitcoind<'a>(
+    progress: (usize, usize),
+    exe_path: &form::Value<String>,
+    started: Option<&Result<std::process::Output, std::io::Error>>,
+) -> Element<'a, Message> {
+    let col_exe_path = Column::new()
+        .push(text("bitcoind executable path:").bold())
+        .push(
+            form::Form::new("/usr/local/bin/bitcoind", exe_path, |msg| {
+                Message::StartInternalBitcoind(message::StartInternalBitcoindMsg::ExePathEdited(
+                    msg,
+                ))
+            })
+            .warning("Please enter correct executable path")
+            .size(20)
+            .padding(10),
+        )
+        .spacing(10);
```

Comment:
I don't think we should ask them about the executable path. We should have a set of paths to be looking at (as you do above using `which`) and fail if there is none. Soon we'll have the option of downloading Bitcoin Core if the executable cannot be found.

Reply:
I've updated this in https://github.com/wizardsardine/liana/commit/09b6c27eabd428e194a97f1e5c2a48c5f543cef6.